### ISSUE TITLE
Flutter WebView loading white page 

### ISSUE THREAD

**Issue by faGalego** [flutter/flutter#61720]
I have an app with a webview (plugin webview_flutter), that load my customer's website.
It is working corretly with his site.

However, I need to use another website that is still in a test enviroment. But this website doesn't have a valid ssl certificate.
When I load this test website the webview turns white and doesn't show the website.

For test, I just changed the WebView's initialUrl to my local website.

Is there an away for ignore the bad certificate validation at least to test?

**Comment by darshankawar** [comment]
Hi @faGalego,
See [this](https://github.com/flutter/flutter/issues/36925) ticket and [this](https://github.com/flutter/flutter/issues/36925#issuecomment-570320289) comment from it.

TL;DR : having valid ssl certificate is a default requirement for all Flutter plugins per design, hence you are seeing blank page.
I am going ahead and closing this ticket, since this is per design.
If you disagree, write in the comments and I'll reopen.
